<mention service name>/README.md

```diff
@@ -0,0 +1,3 @@
+## <mention service name> - Change the name 
+
+Add the required src code under this directory.
```

Comment:
Please remove this line

Add steps to build the applications 

Add steps to build application via docker